package.json

```diff
@@ -47,29 +47,31 @@
     "@rollup/plugin-typescript": "^11.0.0",
     "@types/bsv": "github:chainbow/bsv-types",
     "@types/jest": "^29.0.0",
-    "@types/node": "^20.14.1",
+    "@types/node": "^20.14.2",
     "babel-jest": "^29.0.0",
     "babel-plugin-import": "^1.13.3",
     "better-docs": "^2.7.2",
     "eslint": "^9.4.0",
     "jest": "^29.0.0",
     "jest-fetch-mock": "^3.0.3",
     "jsdoc-babel": "^0.5.0",
-    "prettier": "^3.3.0",
+    "prettier": "^3.3.1",
     "rimraf": "^5.0.7",
     "rollup": "^4.18.0",
     "rollup-plugin-dts": "^6.1.1",
     "rollup-plugin-node-builtins": "^2.1.2",
     "rollup-plugin-node-externals": "^7.0.0",
     "rollup-plugin-polyfill-node": "^0.13.0",
     "ts-node": "^10.9.1",
-    "tslib": "^2.4.0",
+    "tslib": "^2.6.3",
     "typedoc": "^0.25.0",
     "typescript": "^5.0.2"
   },
   "dependencies": {
     "@bsv/sdk": "^1.0.37",
     "@types/rollup-plugin-node-builtins": "^2.1.5",
-    "cross-fetch": "^4.0.0"
+    "cross-fetch": "^4.0.0",
+    "pug": "3.0.3",
+    "pug-code-gen": "3.0.3"
```

Comment:
What? Why?

Reply:
> Pug through 3.0.2 allows JavaScript code execution if an application accepts untrusted input for the name option of the compileClient, compileFileClient, or compileClientWithDependenciesTracked function. NOTE: these functions are for compiling Pug templates into JavaScript, and there would typically be no reason to allow untrusted callers.

and 
*pug-code-gen (npm) <= 2.0.3 3.0.3*

> Pug through 3.0.2 allows JavaScript code execution if an application accepts untrusted input for the name option of the compileClient, compileFileClient, or compileClientWithDependenciesTracked function. NOTE: these functions are for compiling Pug templates into JavaScript, and there would typically be no reason to allow untrusted callers.

Reply:
https://github.com/bitcoin-sv/spv-wallet-js-client/security/dependabot/15
https://github.com/bitcoin-sv/spv-wallet-js-client/security/dependabot/16

Reply:
spv-wallet-js-client does not use `pug` templates

Reply:
but the security alert was there, maybe not directly and some other lib is using it ?

Reply:
`better-docs` uses `pug@2.0.4`. I see two options:
1. Wait until `better-docs` will update their deps
2. Change package to `react-docgen` or `jsdoc`

The last better-docs release was 6 months ago.

Reply:
so should i remove them ? and dismiss alerts, and we will revisit this once better-docs gets upgraded ?

Reply:
i removed both of them .. alerts are dismissed in security .. pls have look again at this pr and let me know next steps